uid2_client/identity_map_client.py

```diff
@@ -0,0 +1,40 @@
+import base64
+import datetime as dt
+from datetime import timezone
+
+from .identity_map_response import IdentityMapResponse
+
+from uid2_client import auth_headers, make_v2_request, post, parse_v2_response
+
+
+class IdentityMapClient:
+    """Client for interacting with UID2 Identity Map services
```

Comment:
note that this class needs to work for EUID as well. Instead of `UID2` in this file we can say `UID2 and EUID` or `UID`

Reply:
I updated it to `UID`